app/_hub/kong-inc/header-cert-auth/_metadata/_index.yml

```diff
@@ -0,0 +1,17 @@
+name: Header Cert Authentication
```

Comment:
Sorry about that, meant to add the preview link. Since 3.8 isn't live yet, the plugin won't show up on this landing page. But you can preview it via https://deploy-preview-7775--kongdocs.netlify.app/hub/kong-inc/header-cert-auth/unreleased/